backend/main.py

```
@@ -143,4 +138,6 @@ def test_email_config():
 
 
 if __name__ == "__main__":
+    # Set DEBUG environment variable for logging system
+    os.environ['DEBUG'] = 'true'
     app.run(debug=True, port=5001)
```

Ruff (0.13.1)

Use of `debug=True` in Flask app detected

(S201)